src/test/scala/is/hail/expr/ir/AggregatorsSuite.scala

```diff
@@ -621,4 +621,11 @@ class AggregatorsSuite {
       FastIndexedSeq(Row(Call2(0, 0), 4L), Row(null, null), Row(null, 2L), Row(Call2(0, 1), 0L), Row(Call2(1, 1), 1L), Row(Call2(0, 2), null)),
       FastIndexedSeq(Call2(0, 1), Call2(1, 1), null))
   }
+
+  @Test def virtualDS() {
+    runAggregator(VirtualDS(), TTuple(TFloat64(), TFloat64()),
+      FastIndexedSeq(Tuple2(1500, 1500), Tuple2(5500, 5500), Tuple2(5600, 5600), Tuple2(9200, 9200), Tuple2(9400, 9400)),
+      FastIndexedSeq(Tuple2(1500, 1500), Tuple2(5500, 5500), Tuple2(9500, 9500)),
```

Comment:
Ah, so Hail type TTuple(TFloat64, TFloat64) is represented by a Row with two Doubles.  So there are two problems here: the Tuple2(...) needs to be Row(...) and the integer literals need to be Double literals.   You can just add a ".0" to the end to make them Doubles.

When I fixed those, it failed because Interpret isn't handling VirtualDS along with the other the other aggregators.